drivers/counter/Kconfig

```diff
@@ -22,6 +22,12 @@ config COUNTER_SHELL
 	help
 	  Enable Shell Commands for Counter and Timer
 
+config COUNTER_64BITS
+	bool
+	help
+	  Set by counter drivers to defines the counter ticks
```

Comment:
`to define` or
```
	  Select this option if the counter driver should use 64-bit
	  tick values for counter_ticks_t instead of 32-bit.
	  This is typically set by drivers that require extended
	  tick ranges beyond the 32-bit limit.
```

Reply:
done